pages/facts.html

```diff
@@ -1,12 +1,45 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <link rel="stylesheet" href="../stylesheets/facts.css">
   <title>Document</title>
```

Comment:
Remember to update the title with something that makes sense for your page! This is what shows up in the browser tab for your page.